pages/common/docker.md

```diff
@@ -2,30 +2,34 @@
 
 > Manage Docker containers and images.
 
-- List of running docker containers:
+- List running docker containers (add -a to see all):
 
 `docker ps`
 
-- List all docker containers (running and stopped):
+- Start or stop a container:
 
-`docker ps -a`
-
-- Start a container:
-
-`docker start {{container}}`
-
-- Stop a container:
-
-`docker stop {{container}}`
+`docker start|stop {{container}}`
 
 - Start a container from an image and get a shell inside of it:
 
 `docker run -it {{image}} bash`
 
-- Run a command inside of an already running container:
+- Run a command inside an already running container:
 
 `docker exec {{container}} {{command}}`
 
 - Remove a container:
 
 `docker rm {{container}}`
+
+- Remove all containers:
```

Comment:
For the `ps` example, you have combined that to a single one saying (add -a to see all). But here, you have mentioned 'remove all containers' in a separate example. I would suggest to either remove the 'remove all' example, or combine it with the previous one. 

Since, your remove all example is not a direct command, rather an aggregate of multiple commands, I would incline towards removing this example altogether. Unless, absolutely required, we strive not to include these kinds of examples.